web/scripts/app.js

```diff
@@ -1600,6 +1609,44 @@ export class ComfyApp {
 		this.lastExecutionError = null;
 		this.runningNodeId = null;
 	}
+
+	switchWorkflow(selected_id) {
+		// If you change the value of this workflow_count, you also need to change the count of `$el("option"~` (ui.js). (count up only)
+		const workflow_count = 10; // also used in getWorkflowNames function
```

Comment:
You should only put this constant in one place and everything should depend on it.